### ISSUE TITLE
Turn off yamllint warnings

### ISSUE THREAD

**Issue by hectorsector** [github/training-kit#496]
## Overview
**TL;DR**
`yamllint` provides some warnings that may not be of any use and more confusing to the learner than they're worth ([example](https://travis-ci.org/githubschool/on-demand-github-pages/builds/208953770#L434)). We should look into their use, and remove them if they serve no purpose.

#### Courses Affected

- [Using GitHub Desktop](https://services.github.com/on-demand/github-desktop/)
- [Using the Command Line](https://services.github.com/on-demand/github-cli/)
- [Repo housing learner activity](https://github.com/githubschool/on-demand-github-pages/)

### How to Contribute
- [ ] Comment on this issue if you'd like to work/collaborate on it! 
- [ ] Fork the [learner activity repository](https://github.com/githubschool/on-demand-github-pages/).
- [ ] Use the [GitHub Flow](https://guides.github.com/introduction/flow/) to make changes to your fork. [This](https://services.github.com/on-demand/intro-to-github/) is a refresher course if you're unsure about how to make a change on GitHub.
- [ ] Push your changes to your repository. 
- [ ] Submit a Pull Request
  - Base Dropdown: githubschool/on-demand-github-pages
  - Compare Dropdown: Your fork

#### Testing Locally
If you'd like to make and test changes locally (and see how they would look if merged), do the following:
- In your command line:
  - [Install Git](https://git-scm.com/)
  - [Install Ruby](https://www.ruby-lang.org/en/documentation/installation/)
  - [ ] Run `script/setup`
  - [ ] Run `script/server`.
    - When successful, the script will initiate a local server at `http://127.0.0.1:4000/on-demand`.

### Questions?
- Leave a comment on this issue! Make sure to use @ mentions if you want a specific person's attention!

**Comment by hectorsector** [comment]
✋ I'm happy to work on this, created the issue so I don't lose track of it.

**Comment by hectorsector** [comment]
With the introduction of teacherbot, I'm going to close this in lieu of upcoming unit tests via the tool. If anyone wants to try their ✋ at this, please feel free to reopen and introduce a PR.
